src/btree/bt_vrfy.c

```diff
@@ -658,16 +658,22 @@ __verify_tree(
             ++vs->depth;
             ret = __wt_page_in(session, child_ref, 0);
 
-            /*
-             * If configured, continue traversing through the pages of the tree even after
-             * encountering errors reading in the page.
-             */
-            if (vs->read_corrupt && ret != 0) {
+            if (ret != 0) {
+                if (vs->dump_address)
+                    WT_TRET(__wt_msg(session,
+                      "%s Read failure while accessing a page from the column internal page (ret = "
+                      "%d).",
+                      __verify_addr_string(session, ref, vs->tmp1), ret));
```

Comment:
I think `ref` here should be `child_ref`. We want to print the address from the same page (i.e., WT_REF) that we tried to read in the call to `__wt_page_in()` above.

I noticed this because when I ran this on a corrupt file, the address in the dump output didn't match the address printed by this message.

Here's example dump output:
```
[1705627834:608570][9984:0x7ffff7fd5780], wt, file:collection-50-2911778453470923261.wt, WT_SESSION.verify: [WT_VERB_DEFAULT][ERROR]: int __bm_corrupt_dump(WT_SESSION_IMPL *, WT_ITEM *, uint32_t, wt_off_t, uint32_t, uint32_t), 89: {0: 356352, 12288, 0x443481c0}: (chunk 10 of 12): 63 09 fe 63 09 
```
The numbers in braces -- `{0: 356352, 12288, 0x443481c0}` are the same values printed by `__verify_addr_string()`